build/Targets/GenerateAssemblyInfo.targets

```diff
@@ -19,6 +19,9 @@
     <AssemblyVersionAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute">
       <_Parameter1>$(NuGetPerBuildPreReleaseVersion)</_Parameter1>
     </AssemblyVersionAttribute>
+    <AssemblyVersionAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
```

Comment:
I think description is assumed to be user-visible text. The AssemblyInformationalVersionAttribute right above is an arbitrary string, put it there?